core/store/migrations/migration1568390387/migrate.go

```diff
@@ -1,18 +1,33 @@
 package migration1568390387
 
 import (
+	"time"
+
 	"github.com/jinzhu/gorm"
 	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/store/assets"
 )
 
+type Encumbrance struct {
+	ID                     uint         `gorm:"primary_key;auto_increment"`
+	Payment                *assets.Link `gorm:"type:varchar(255)"`
+	Expiration             uint64
+	EndAt                  time.Time
```

Comment:
Should this be `null.Time`?

Reply:
It looks like the underlying type is `time.Time` with an additional boolean. It seems to work, but maybe we should investigate switch `models.AnyTime` to be an alias of `null.Time`?

Reply:
I mean if EndAt is null, what is a null time.Time? Is it the 1970 epoch? Is that okay?

Reply:
It is ok, because it's an invalid `endAt` and might as well be null. I'm open to changing it, but this is all behind the dev flag. I think we should get this release out and then revisit this in another PR. Captured it in a story here: https://www.pivotaltracker.com/story/show/169661678

Reply:
Cool, just checking.